utils/utils_test.go

```diff
@@ -31,3 +31,11 @@ var _ = Describe("ProjectResource", func() {
 		Expect(ProjectResource("fake-project-1")).To(Equal("projects/fake-project-1"))
 	})
 })
+
+var _ = Describe("SplitExtraFilter", func() {
```

Comment:
Could you add a test for the case where there's no `:` in the filter as well please?

Reply:
No problem. Also added one for the incomplete/missing filter.
I don't have much experience with ginkgo but it seems that with v2 (or extensions/table) we could use `DescribeTable` for nicer parametrized tests. But I didn't want to introduce/change dependencies here.

Reply:
Thank you! The testing framework is rather outdated 😬 . I imagine we would probably look to remove the library in favor of more basic std lib based tests but never really got the time to do it.